src/mbgl/renderer/fill_bucket.cpp

```diff
@@ -129,6 +130,14 @@ void FillBucket::drawElements(FillShader& shader,
     for (auto& group : triangleGroups) {
         group.getVAO(shader, paintMode).bind(
             shader, *vertexBuffer, *triangleIndexBuffer, vertex_index, context);
+        if (context.elementBuffer.getCurrentValue() == 0) {
```

Comment:
A note that this call doesn't actually query the state; it just returns the internally tracked state, which might be out of sync with the OpenGL state (although it shouldn't be, ideally). To query the actual OpenGL state, use `gl::value::BindElementBuffer::Get()`